catalogue_of_life/schema.tql

```diff
@@ -0,0 +1,182 @@
+define
+
+# Common attributes
+web-link sub attribute, value string;
+
+# Taxa
+
+taxon-id sub attribute, value string;
+
+taxon-status sub attribute,
+    value string,
+    regex "^(accepted|provisionally accepted|misapplied|synonym|ambiguous synonym)$";
+
+scientific-name sub attribute, value string;
+taxon-authorship sub attribute, value string;
+
+taxon-rank sub attribute, value string;
+taxon-notho sub attribute,
+    value string,
+    regex "^(|generic|specific|infraspecific)";
+nomenclatural-code sub attribute,
+    value string,
+    regex "^(|bacterial|botanical|virus|zoological)$";
+
```

Comment:
what's your thinking behind where the newlines are placed between attribute definitions?